flink-yarn/src/main/java/org/apache/flink/yarn/YarnResourceManager.java

```diff
@@ -514,8 +514,7 @@ private void requestYarnContainer() {
 		resourceManagerClient.addContainerRequest(getContainerRequest());
 
 		// make sure we transmit the request fast and receive fast news of granted allocations
-		resourceManagerClient.setHeartbeatInterval(FAST_YARN_HEARTBEAT_INTERVAL_MS);
-
+		resourceManagerClient.setHeartbeatInterval(containerRequestHeartbeatIntervalMillis);
```

Comment:
Hi, @tillrohrmann  @HuangZhenQiu, `resourceManagerClient.setHeartbeatInterval(containerRequestHeartbeatIntervalMillis);` need set the value when `requestYarnContainer` called each time?  thanks.

Reply:
It is not optimal but it should also not hurt too much. Moreover, at the moment this method is the common place which is called for every container request. Changing it should be out of scope for this PR.

Reply:
> It is not optimal but it should also not hurt too much. Moreover, at the moment this method is the common place which is called for every container request. Changing it should be out of scope for this PR.

ok, I see.